fwup_include/fwup-common.conf

```diff
@@ -33,21 +33,25 @@ define(ROOTFS, "${NERVES_SYSTEM}/images/rootfs.squashfs")
 # | Firmware configuration data|
 # | (formatted as uboot env)   |
 # +----------------------------+
-# | p0*: Boot A (FAT32)        |
+# | p1: Boot A (FAT32)         |
 # | kernel8.img, start4.elf,   |
 # | config.txt, etc.           |
 # +----------------------------+
-# | p0*: Boot B (FAT32)        |
+# | p2: Boot B (FAT32)         |
 # +----------------------------+
-# | p1*: Rootfs A (squashfs)   |
+# | p0: Autoboot (FAT32)       |
```

Comment:
Is this only for storing the autoboot that we upgraded from so we can use it to revert?

I would have expected that to be handled by the autoboot.txt being packed into the fwup-ops .fw file. Is the concern that people might break compatibility or is there something I'm not seeing?

Reply:
Ah, then I haven't read it in enough detail recently enough. Brain has bitrotted.